ci/Dockerfile

```diff
@@ -1,62 +1,140 @@
-FROM a12e/docker-qt:5.14-gcc_64
+ARG QT_VERSION=5.15.2
+
+# QT Installation Image --------------------------------------------------------
+FROM ubuntu:18.04 AS qt-install
+
+ARG QT_VERSION
+
+RUN apt update && apt full-upgrade -y && apt install -y --no-install-recommends \
+    sudo python3 python3-pip \
+    && apt-get -qq clean
+
+RUN python3 -m pip install setuptools
+
+RUN chmod -R 777 /opt
+
+RUN groupadd -r user && useradd --create-home --gid user user && echo 'user ALL=NOPASSWD: ALL' > /etc/sudoers.d/user
+
+
+COPY scripts/install-qt.sh /home/user/
+RUN chown user:user /home/user/install-qt.sh
+RUN chmod a+x /home/user/install-qt.sh
```

Comment:
Not sure if this makes sense as a separate script. You'r running 4 commands in a Dockerfile in total, all just to call a script that does this:
https://github.com/status-im/status-desktop/blob/fb28070e340bafc5258a6228dd66b2cfe326c8b0/ci/scripts/install-qt.sh#L12-L14
Why not just do that here directly? This only obscures what's actually happening.